en/integrations/plugin-packs/procedures/applications-databases-elasticsearch.md

```diff
@@ -1,38 +1,141 @@
+| status                   | Status of the Elasticsearch cluster      |         |
+| shards_active            | Number of active shards                  |         |
+| shards_unassigned        | Number of unassigned shards              |         |
+| docs_count               | Number of docs                           |         |
+| size_in_bytes_primaries  | Size of all primary shards               |  bytes  |
+| size_in_bytes_total      | Total size of all shards assigned        |  bytes  |
+
+<!--license-->
+
+| Metric name        | Description                              | Unit    |
+| :----------------- | :--------------------------------------- | :------ |
+| type               | Licence type                             |         |
+| status             | Licence status                           |         |
+| issued_to          | Licence owner                            |         |
+| issue_date         | Licence issue date                       |  date   |
+
+<!--list-indices-->
+
+| Metric name        | Description                             | Unit    |
+| :----------------- | :-------------------------------------- | :------ |
+| name               | Indice name                             |         |
+| status             | Indice status                           |         |
+
+<!--list-nodes-->
+
+| Metric name        | Description                             | Unit    |
+| :----------------- | :-------------------------------------- | :------ |
+| name               | Node name                               |         |
+| host               | Node host                               |         |
+| ip                 | Node ip                                 |   ip    |
+
+<!--node-statistics-->
+
+| Metric name       | Description                                           | Unit    |
+| :---------------- | :---------------------------------------------------- | :-----  |
+| display           | Name of the Elasticsearch node                        |         |
+| indices_count     | Number of indices in the node                         |         |
+| heap_used_percent | Percentage of memory currently in use by the heap     |    %    |
+| heap_used_in_bytes| Amount of memory available for use by the heap        |  bytes  |
+| heap_max_in_bytes | Maximum amount of memory available for use by the heap|  bytes  |
+| available_in_bytes| Total number of bytes available                       |  bytes  |
+| total_in_bytes    | Total size of all file stores                         |  bytes  |
+| docs_count        | Number of documents inside the indice                 |         |
+| size_in_bytes     | Total size of all shards assigned to the node         |  bytes  |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisites 
+
+In order to monitor an Elasticsearch cluster, it must be prepared acccording to Elasticsearch's official documentation: https://www.elastic.co/guide/en/elasticsearch/reference/7.8/monitor-elasticsearch-cluster.html
+
```

Comment:
Please explain protocol and default port used and that the Centreon Poller should be able to communicate with the Elastic node through this network flow.

Reply:
Not sure if it's what you wanted but i added it. After all the port and the protocol can be adjusted in the centreon web interface as well, so it's not that mandatory, it just have to be the same on both ends.